composer.json

```diff
@@ -212,6 +212,7 @@
         "furf/jquery-ui-touch-punch": "dev-master",
         "geocoder-php/open-cage-provider": "^4.4",
         "massgov/mayflower-artifacts": "11.7.0",
+        "mikeyp/google_tag": "dev-8.x-1.x#3caa4f0b7f008576936ab7501d483bfcf58a4764",
```

Comment:
why is this package added?

Reply:
It seems that was coming from resolving merge conflicts when `develop` was merged in.  It was removed since it's not related this ticket's change.